Formula/mtr.rb

```diff
@@ -1,10 +1,9 @@
 class Mtr < Formula
   desc "'traceroute' and 'ping' in a single tool"
   homepage "https://www.bitwizard.nl/mtr/"
-  url "https://github.com/traviscross/mtr/archive/v0.93.tar.gz"
-  sha256 "3a1ab330104ddee3135af3cfa567b9608001c5deecbf200c08b545ed6d7a4c8f"
+  url "https://github.com/traviscross/mtr/archive/v0.94.tar.gz"
+  sha256 "ea036fdd45da488c241603f6ea59a06bbcfe6c26177ebd34fff54336a44494b8"
   license "GPL-2.0"
```

Comment:
This should be "GPL-2.0-only" instead, according to its license header (in https://github.com/traviscross/mtr/blob/master/ui/asn.c, for example)